src/lib.rs

```diff
@@ -935,10 +923,10 @@ fn parse_names(raw_face: &ttf_parser::RawFace) -> Option<(Vec<(String, Language)
         if let Some(index) = families
             .iter()
             .position(|f| f.1 == Language::English_UnitedStates)
-            .filter(|index| *index > 0)
         {
-            let v = families.remove(index);
-            families.insert(0, v)
+            if index != 0 {
```

Comment:
I quickly moved the index check down, as otherwise it might skip over the first English name and still swap the second one in.

Reply:
Are you sure? The idea is that if an English name isn't the first one (which happens), then we have to move to the first place.

Reply:
Oh wait, I thought that was Iterator::filter, not Option::filter. My if basically does the same as your previous Option::filter call then, so there's no change in behavior. I can revert that part if you want. (Though arguably the filter call is a little confusing as it's part of an iterator chain and not the actual Iterator::filter method)

Reply:
Sure, let's leave as `if` condition for readability.